packages/mariadb-10.11-mroonga/Rakefile

```diff
@@ -6,7 +6,9 @@ class MariaDB1011MroongaPackageTask < MroongaPackageTask
   end
 
   def enable_apt?
```

Comment:
`apt_targets_default`

Reply:
Thank you for your comment.
I've fixed it.